tests/test_app/TestApp/Policy/StringResolver.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         1.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace TestApp\Policy;
+
+use Authorization\Policy\Exception\MissingPolicyException;
+use Authorization\Policy\ResolverInterface;
+use Cake\Core\App;
+use Cake\Datasource\EntityInterface;
+use Cake\Datasource\QueryInterface;
+use Cake\Datasource\RepositoryInterface;
+
+/**
+ * Very simple policy resolver that accepts string policy names.
+ */
+class StringResolver implements ResolverInterface
+{
+    /**
+     * Get a policy for a string.
+     *
+     * @param string $resource The resource.
+     * @return object
```

Comment:
According to used class (constructor of it), this is `\Authorization\Policy\ResolverInterface`

Reply:
The @return doc? That was copied directly from the OrmResolver.

Reply:
Being stricter seems like a good idea, but is outside the scope of this PR, I think. Addresses a different issue.

Reply:
I don't think Policies have a common interface though.

Reply:
I think he was suggesting that they should? Which, like I said, would be a separate issue.

Reply:
Yes it is a separate conversation.